saleor/graphql/schema.graphql

```diff
@@ -1254,8 +1254,10 @@ enum OrderStatus {
 }
 
 enum OrderStatusFilter {
-  READY_TO_FULFILL
```

Comment:
So we can't filter by READY_TO_FULFILL and READY_TO_CAPTURE, right? But i think storefont 2.0 is using them?

Reply:
Ah, you're right. It probably does use them. I'll add them back for backward compatibility reasons, but sadly I can't mark that in the API.

Reply:
Or actually, I'll revert that change and we can come back to it at a later stage. Nothing uses it anyway.